sky/cli.py

```diff
@@ -1917,6 +1917,32 @@ def start(
     if not to_start:
         return
 
+    # Checks for reserved clusters (spot controller).
+    reserved, non_reserved = [], []
+    for name in to_start:
+        if name in backend_utils.SKY_RESERVED_CLUSTER_NAMES:
+            reserved.append(name)
+        else:
+            non_reserved.append(name)
+    if reserved and non_reserved:
```

Comment:
Questioning this design choice here, why must Spot controller be started on its own? 

If I run `sky start hello bye spot_controller -i 5`, it should start `hello` and `bye`, but `spot_controller` should error.

Reply:
Sounds good to me.